mbtb_app/web_portal/mbtb_portal/api/controllers/admin/suspend-user-form.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+
+
+  friendlyName: 'Suspend user form',
+
+
+  description: '',
+
+
+  inputs: {
+
+  },
+
+
+  exits: {
+    success:{
+      viewTemplatePath: 'pages/suspend_user_form',
+      description: 'return view for suspending user reason',
+      locals: {
+        layout: 'layouts/admin_layout'
+      }
+    }
+  },
+
+  fn: async function (inputs, exits) {
+
+    return exits.success();
```

Comment:
6 -> We need to save those received ids, in other words we need to pass it to next action/screen. So here we'll pass it to ejs view.

```
return exits.sucess({request_ids: inputs.request_ids})
```